docs/pages/experiments/joy/app-new-theme.tsx

```diff
@@ -261,9 +261,18 @@ export const newTheme = extendTheme({
         },
       },
     },
+    JoySheet: {
+      styleOverrides: {
+        root: ({ ownerState }) => ({
+          ...(ownerState.variant === 'solid' && {
+            backgroundColor: '#000',
+          }),
+        }),
+      },
+    },
     JoyCard: {
       styleOverrides: {
```

Comment:
@siriwatknp is there a way to customize styleOverrides according to the theme (light or dark) ?

Reply:
Yes, you can use `theme.getColorSchemeSelector('dark')` to apply the styles for dark mode.

```js
JoyCard: {
  styleOverrides: {
    root: ({ theme, ownerState }) => ({
      …custom styles for default mode,
      [theme.getColorSchemeSelector('dark')]: {
        …custom styles for dark mode
      }
    })
  }
}
```